documentation/modules/post/windows/gather/credentials/windows_sam_hivenightmare.md

```diff
@@ -0,0 +1,59 @@
+## Vulnerable Application
+
+### Description
+
+Due to mismanagement of SAM and SYSTEM hives in Windows 10, it is possible for an unprivileged
+user to read those files. But, as they are locked while Windows is running we are not able
+to read them directly. The trick is to take advantage of Volume Shadow Copy, which is generally
+enabled, to finally have a read access. Once SAM and SYSTEM files are successfully dumped and
+stored in `store_loot`, you can dump the hashes with some external scripts like secretsdump.py
+
+### Installation
+
+VSS is probably already enabled on your Windows 10, if you want to be sure you can follow the steps below :
+
+- Go to `System Properties GUI`
+- Go to `System Protection`tab
+- See in `Protection Settings` list if your drive protection is enabled
+
+Be aware that you will need Administrator privileges to follow those steps.
+You can read more [here](https://isc.sans.edu/diary/Summer+of+SAM+-+incorrect+permissions+on+Windows+1011+hives/27652).
+
+## Verification Steps
+
+
+1. Start `msfconsole`
+2. `use post/windows/gather/credentials/windows_sam_hivenightmare`
+3. `set NBRE_ITER <number>` to specify the number of iterations on file index (default is 10)
+4. `set FILE_INDEX <number>` optionally if you want to target a specific file index instead of iterating on all indexes in a range
+5. `run` the module to exploit the vulnerability and potentially leak SAM and SYSTEM files
+
+## Options
+
+### NBRE_ITER
+
+Set NBRE_ITER to specify the number of iterations on Shadow Copy file index.
+Windows is saving those files under the volume name `HarddiskVolumeShadowCopy<index>`.
+By default, this module is bruteforcing that `index` value by trying all values between 0 and 10, which you can change if needed.
+
+### FILE_INDEX
+
+Set FILE_INDEX if you want to target a specific index instead of the default behaviour which bruteforces all indexes in a given range.
+
+## Scenarios
+
+This module was successfully tested on Windows 10 20H2.
+See the following output :
```

Comment:
Minor suggestion:

```suggestion
See the following output:
```

Also would you be able to please add a note about how secretsdump.py should be used and maybe even the syntax to then extract the goods out of the SAM file? That could really help folks that are looking to use this module.